### DISCUSSION TITLE
More than one pinned post

### DISCUSSION THREAD

**Discussion by leewynne** [forem/forem#16360] [Feature Requests]
Would love to have at least 2 pinned posts.

I feel certain communities have more that one relevant post which is of high importance.

**Comment by ildi-x** [comment]
I think it would be valuable to allow for more than 1 pinned post. This has me wondering if admins can pin as many posts as they want or if there should be a set limit. How flexible should this be? Im sort of leaning towards a limit of 2 pinnned posts.

**Reply by ellativity** [reply]
I agree that we shouldn't have unlimited pinning. Although from an Admin perspective I can see the use value there, I think it could lead to a messy user experience. Especially for users of chromebooks or other smaller landscape-oriented displays, having too many pinned posts means scrolling too long to even get into the feed.
